tutorials/plot_visualize_evoked.py

```diff
@@ -115,6 +115,30 @@
                         ts_args=ts_args, topomap_args=topomap_args)
 
 ###############################################################################
+# Sometimes, you may want to compare two conditions at a selection of sensors,
+# or e.g. for the Global Field Power. For this, you can use the function
+# `mne.viz.plot_compare_evokeds`. The easiest way is to create a  Python
+# dictionary, where the keys are condition names and the values are
+# `mne.Evoked` objects. If you provide lists of `mne.Evoked` objects, such as
+# those for multiple subjects, the grand average is plotted, along with a
+# confidence interval band - this can be used to contrast conditions for
+# a whole experiment.
+# First, we load in the evoked objects into a dictionary, setting the keys to
+# '/'-separated tags. Then, we plot with `mne.viz.plot_compare_evokeds`.
+# The plot is styled with dictionary arguments, again using "/"-separated tags.
+# We plot a MEG channel with a strong auditory response.
+conditions = ["Left Auditory", "Right Auditory", "Left visual", "Right visual"]
+evoked_dict = dict()
+for condition in conditions:
+    evoked_dict[condition.replace(" ", "/")] = mne.read_evokeds(
+        fname, baseline=(None, 0), proj=True, condition=condition)
+print(evoked_dict)
+colors = dict(Left="Crimson", Right="CornFlowerBlue")
+linestyles = dict(Auditory='-', visual='--')
+mne.viz.plot_compare_evokeds(evoked_dict, picks=200,
```

Comment:
Still, you could argue it should rather be one figure with multiple axes, like evoked.plot, rather than multiple figure, like evoked.plot_joint. Opinion?